src/refract/SerializeVisitor.cc

```diff
@@ -54,6 +55,23 @@ namespace refract
             return array;
         }
 
+        IElement::renderFlags SelectSerializationType(const IElement& e, bool exportSourceMap) {
+
+            if (exportSourceMap) {
+                IElement::MemberElementCollection::const_iterator it = e.attributes.find("sourceMap");
+                // there is sourceMap in attributes
```

Comment:
Btw, an "objective-c-ism" to turn this comment into a property:

```cpp
bool attributesIncludesSourceMap = it != e.attributes.end();

if (attributesIncludesSourceMap) {
    IElement::rFull
}
```

I think this is a great way to write more readable code without comments.